ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/signup/clientlibs/js/signup.js

```diff
@@ -133,11 +133,13 @@ function displayError(errorhtml, formElmName) {
 		  displayError(errorhtml, 'terms');
 		  return false;
 	  }
-	if(validateRecaptcha() == false) {
-          errorhtml = '<span>please verify you are human!<span>';
-          displayError(errorhtml, 'terms');
-          return false;
-      }
+	  if(value){
```

Comment:
@nitheshkumar-hcl : same as above comment

Reply:
Changed as per the review comments